Practice.ipynb

```diff
@@ -17,6 +17,50 @@
     "\n",
     "Reproduce KB's results"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "76482792",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ImportError",
+     "evalue": "cannot import name 'molecules' from 'ase.build' (/opt/anaconda3/envs/MLenv/lib/python3.9/site-packages/ase/build/__init__.py)",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[5], line 11\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mdscribe\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mdescriptors\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m MBTR\n\u001b[1;32m     10\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mase\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mio\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m read\n\u001b[0;32m---> 11\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mase\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mbuild\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m molecules\n\u001b[1;32m     12\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mase\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m Atoms\n\u001b[1;32m     13\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mnumpy\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mlinalg\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m eig\n",
+      "\u001b[0;31mImportError\u001b[0m: cannot import name 'molecules' from 'ase.build' (/opt/anaconda3/envs/MLenv/lib/python3.9/site-packages/ase/build/__init__.py)"
+     ]
+    }
+   ],
+   "source": [
+    "#import modules\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from pymatgen.io.ase import AseAtomsAdaptor as AAA\n",
+    "from matminer.featurizers import structure as sf\n",
+    "from dscribe.descriptors import CoulombMatrix as CM\n",
+    "from dscribe.descriptors import SOAP \n",
+    "from dscribe.descriptors import MBTR\n",
+    "from ase.io import read\n",
+    "from ase.build import molecules\n",
```

Comment:
Hi @stfalxndria, I think this generates an error. It should be 

from ase.build import molecule

Could you please fix the issue and let me know when it's done so I can merge the PR?

Cheers,

Bruno